paybybank-us/src/main/java/com/adyen/checkout/paybybankus/internal/DefaultPayByBankUSDelegate.kt

```diff
@@ -125,4 +129,14 @@ internal class DefaultPayByBankUSDelegate(
         removeObserver()
         analyticsManager.clear(this)
     }
+
+    override fun getBrandList(): List<LogoTextItem> {
+        return listOf(
+            LogoItem(PayByBankUSBrandLogo.US_1.path, componentParams.environment),
+            LogoItem(PayByBankUSBrandLogo.US_2.path, componentParams.environment),
+            LogoItem(PayByBankUSBrandLogo.US_3.path, componentParams.environment),
+            LogoItem(PayByBankUSBrandLogo.US_4.path, componentParams.environment),
+            TextItem(R.string.checkout_pay_by_bank_us_more),
```

Comment:
Consider using an icon for this instead of text, if a merchant is using a custom font it might make this look ugly/different

Reply:
I am not so sure about this. I get your point but I think leaving it as text allows merchants to use some other text such as `and more` or `+` instead of `+ more`. I prefer it staying this way unless you are strongly against it.